_news/2019-04-10-config-validator.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Forseti Config Validator Efforts
+author: Alex Sung
+---
+Today we’re excited to announce Forseti Config Validator, the newest addition to the 
+Forseti Security toolkit. Config Validator helps cloud admins put guardrails in place 
+to protect against misconfigurations in Google Cloud Platform environments. This allows 
+developers to move quickly, and gives security and governance teams the ability to 
+enforce security at scale.
+
+**How it works**  
```

Comment:
Capitalize the first letter of each word, it will look more professional in the header.
`**How It Works`
Please do this throughout.

Reply:
updated